src/masternodes/validation.cpp

```diff
@@ -2444,7 +2444,13 @@ static void ProcessEVMQueue(const CBlock &block, const CBlockIndex *pindex, CCus
         RevertFailedTransferDomainTxs(failedTransactions, block, chainparams.GetConsensus(), pindex->nHeight, cache);
     }
 
-    cache.AddBalance(minerAddress, {DCT_ID{}, static_cast<CAmount>(blockResult.miner_fee / CAMOUNT_TO_GWEI)});
+    if (pindex->nHeight >= chainparams.GetConsensus().ChangiIntermediateHeight4) {
+        cache.AddBalance(Params().GetConsensus().burnAddress, {DCT_ID{}, static_cast<CAmount>(blockResult.total_burnt_fees / CAMOUNT_TO_GWEI)});
+        cache.AddBalance(minerAddress, {DCT_ID{}, static_cast<CAmount>(blockResult.total_priority_fees / CAMOUNT_TO_GWEI)});
```

Comment:
If EVM returns Sats then we can remove CAMOUNT_TO_GWEI.

Reply:
Resolved in commit ee66ff567